apps/prairielearn/src/pages/instructorGradingJob/instructorGradingJob.html.ts

```diff
@@ -0,0 +1,218 @@
+                                >
+                              </td>
+                              <td>
+                                Contains all files necessary for grading; this is what is mounted to
+                                <code>/grade</code> when your job is run.
+                              </td>
+                            </tr>
+                            <tr>
+                              <td>
+                                <a
+                                  href="${resLocals.urlPrefix}/grading_job/${gradingJobQueryResult
+                                    .grading_job.id}/file/archive.tar.gz"
+                                  >archive.tar.gz</a
+                                >
+                              </td>
+                              <td>
+                                A snapshot of <code>/grade</code> after your job has been executed.
+                              </td>
+                            </tr>`
+                        : ''}
+                      <tr>
+                        <td>
+                          <a
+                            href="${resLocals.urlPrefix}/grading_job/${gradingJobQueryResult
+                              .grading_job.id}/file/results.json"
+                            >results.json</a
+                          >
+                        </td>
+                        <td>
+                          Contains the PrairieLearn-generated results, which includes the contents
+                          of your
+                          <code>results.json</code> as well as some PrairieLearn metadata.
+                        </td>
+                      </tr>
+                      <tr>
+                        <td>
+                          <a
+                            href="${resLocals.urlPrefix}/grading_job/${gradingJobQueryResult
+                              .grading_job.id}/file/output.log"
+                            >output.log</a
+                          >
+                        </td>
+                        <td>
+                          Contains the raw output from stdout/stderr for your job. Lines beginning
+                          with
+                          <code>container &gt;</code> are from your container; the rest are
+                          diagnostic logs from PrairieLearn.
+                        </td>
+                      </tr>
+                    </tbody>
+                  </table>
+                </div>
+              `
+            : ''}
+
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">Job Output</div>
+            <div class="card-body">
+              ${gradingJobQueryResult.grading_job.s3_bucket &&
+              gradingJobQueryResult.grading_job.s3_root_key
+                ? html`
+                    <script>
+                      const outputUrl = document.getElementById('job-output').dataset.outputUrl;
```

Comment:
All this code will need to be wrapped in `$(() => { ... })` to ensure that it doesn't run until after the document is completely loaded. Documentation here: https://learn.jquery.com/using-jquery-core/document-ready/

As it stands, the element with ID `job-output` doesn't exist at the time that this script runs.

Reply:
Resolved in commit 4dc36e8.

Reply:
This was not resolved; your diff declares a function but doesn't do anything with it. Please review my suggestion and the linked documentation again, specifically the code block starting with `// Shorthand for $( document ).ready()`.

Reply:
You are right, sorry about that. I have now updated code to be wrapped in `$(function() => {...})` per the documentation.